core/java/com/google/instrumentation/stats/RpcConstants.java

```diff
@@ -159,7 +159,7 @@
           Arrays.asList(RPC_CLIENT_METHOD));
   public static final DistributionViewDescriptor RPC_CLIENT_RESPONSE_BYTES_VIEW =
       DistributionViewDescriptor.create(
-          "/rpc/client/response_bytes",
+          "rpc client response_bytes",
```

Comment:
No, I think this is still open for discussion - I've made the name here consistent with the other names but they may be changed.